steps/android-instrumented-test/0.1.0/step.yml

```diff
@@ -0,0 +1,65 @@
+title: Android Instrumented Test
+summary: Runs Instrumented tests on an existing APK
+description: Runs Instrumented tests on an existing APK
+website: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test
+support_url: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test/issues
+published_at: 2022-06-27T11:57:49.870018-05:00
+source:
+  git: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test.git
+  commit: 03ef48593dedade39df09eb723512649c9d5bd67
+host_os_tags:
```

Comment:
`host_os_tags` is deprecated can be removed.

Reply:
OK, I will remove it